<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecurityDialog</class>
 <widget class="QDialog" name="SecurityDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LICENSE PLATE PROPERTY</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>960</width>
     <height>660</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="buttonNewPassword">
    <property name="geometry">
     <rect>
      <x>827</x>
      <y>190</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNewPassword">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>428</x>
      <y>190</y>
      <width>400</width>
      <height>110</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>100</horstretch>
      <verstretch>100</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>50</height>
     </size>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelSecurity">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>25</y>
      <width>910</width>
      <height>150</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Please input new password.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClose">
    <property name="geometry">
     <rect>
      <x>738</x>
      <y>547</y>
      <width>200</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSave">
    <property name="geometry">
     <rect>
      <x>538</x>
      <y>547</y>
      <width>200</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditConfirmPassword">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>428</x>
      <y>299</y>
      <width>400</width>
      <height>110</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>100</horstretch>
      <verstretch>100</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>50</height>
     </size>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonConfirmPassword">
    <property name="geometry">
     <rect>
      <x>827</x>
      <y>299</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNewPassword">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>190</y>
      <width>404</width>
      <height>110</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>New Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelConfirmPassword">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>299</y>
      <width>404</width>
      <height>110</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Confirm Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSystemLock">
    <property name="geometry">
     <rect>
      <x>427</x>
      <y>408</y>
      <width>510</width>
      <height>110</height>
     </rect>
    </property>
    <property name="text">
     <string>On/Off</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSystemLock">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>408</y>
      <width>404</width>
      <height>110</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>System Lock</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>428</x>
      <y>408</y>
      <width>508</width>
      <height>110</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>826</x>
      <y>190</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>826</x>
      <y>299</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>lineEditNewPassword</zorder>
   <zorder>labelSecurity</zorder>
   <zorder>buttonClose</zorder>
   <zorder>buttonSave</zorder>
   <zorder>lineEditConfirmPassword</zorder>
   <zorder>labelNewPassword</zorder>
   <zorder>labelConfirmPassword</zorder>
   <zorder>buttonSystemLock</zorder>
   <zorder>labelSystemLock</zorder>
   <zorder>buttonConfirmPassword</zorder>
   <zorder>buttonNewPassword</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
